### PR TITLE
log: Add --log-format flag #2424

### PR DESCRIPTION
**Pull request by dcpu**
Show microseconds for easier finding performance bottlenecks.

### CODE REVIEW

**Comment by dcpu** [comment]
Updated to correspond to #2424
https://github.com/ncw/rclone/pull/2494/commits/d470065ce7c60a4bcf1adc658b7872efa603bfeb

File: fs/log/log.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ import (
 // Flags
 var (
 	logFile        = flags.StringP("log-file", "", "", "Log everything to this file")
+	logFormat      = flags.StringP("log-format", "", "date,time", "")
```
[the remark this answers is not in the document]
Comment (dcpu) [reply fs/log/log.go]: It is not easy for me to write a document. (English is not my first language.) That's why I left it blank. Any help would be greatly appreciated.

**Comment by ncw** [comment]
Docs look good thank you :-)

Can you run `go fmt fs/log/log.go` to get the CI to run clean please?

**Comment by ncw** [comment]
errchk is complaining about `fs/log.go:73:12:	log.Output(4, text)`

I suggest we ignore the error since there is nowhere to log it with

    _ = log.Output(...)

**Comment by ncw** [comment]
That looks great now - thank you.

I've merged it now.

**Pull request merged**